src/base/static/scss/_basic.scss

```diff
@@ -196,4 +196,46 @@ img {
 
 .icon-muted {
     opacity: 0.75;
+}
+
+/* Overrides for Quill rich text editor styles. We set these to prevent text 
+   styles within the rich text editor from rendering in a different style than
+   they do in place detail views
+*/
+
+.ql-editor {
+    h1, h2 {
+        font-size: 2em !important;
+        font-family: "Roboto", sans-serif !important;
```

Comment:
Seems like the font-family and margin-bottom here is common to all, factor it out.